utils/managed_bytes.hh

```diff
@@ -462,6 +462,17 @@ public:
         return fragment_iterator();
     }
     size_t size_bytes() const noexcept { return size(); }
+
+    size_t hash() const noexcept {
+        if (!_next_fragments) {
```

Comment:
`with_linearized()` does this also, so no need to do this here.

Reply:
right, thanks.